<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PotokAdder</class>
 <widget class="QWidget" name="PotokAdder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1224</width>
    <height>773</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>1211</width>
     <height>751</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1171</width>
      <height>721</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1169</width>
       <height>719</height>
      </rect>
     </property>
     <widget class="QLineEdit" name="Potok_name">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>25</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Введите название потока</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Add_line">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>163</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить новую строку</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Create_potok">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>243</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Создать поток</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>25</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Введите название групп/ы</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Group_name">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>80</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Error_Box">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>85</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ошибка</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
